Sources/OktaIdx/Extensions/IDXClient+Response.swift

```diff
@@ -184,6 +184,11 @@ public extension IDXClient {
         
         /// The type of this token.
         @objc public let tokenType: String
+        
+        @objc public enum RevokeType: Int {
+            case refreshToken
+            case accessAndRefreshToken
```

Comment:
What about ID token?

Reply:
The [revoke API only provides these two options](https://developer.okta.com/docs/guides/revoke-tokens/revokeatrt/), from what I can tell.